src/twisted/application/twist/test/test_twist.py

```diff
@@ -45,12 +45,12 @@ def patchInstallReactor(self):
         """
         self.installedReactors = {}
 
-        def installReactor(name):
+        def installReactor(_, name):
             reactor = MemoryReactor()
             self.installedReactors[name] = reactor
             return reactor
 
-        self.patch(_options, "installReactor", installReactor)
+        self.patch(TwistOptions, "installReactor", installReactor)
```

Comment:
Not feeling particularly strong about this one, as we're patching the object being tested itself. I'm open to suggestions.